java-optional/src/solutions/java/none/cvg/optional/TestSolution1OptionalCreationAndFetchingValues.java

```diff
@@ -159,12 +159,12 @@ public void getValueFromOptionalForNonNullValue() {
          * DONE:
          *  Replace the "null" to get the Optional has a null value.
          *  Verify that the call throws a NoSuchElementException
-         *  Check API: java.util.Optional.isPresent()
+         *  Check API: java.util.Optional.orElseThrow()
          */
         assertThrows(NoSuchElementException.class, () -> {
 
             assertNotEquals(10,
-                    anotherOptionalInteger.get(),
```

Comment:
This should remain "get()" and not "orElseThrow()", since the latter has not yet been introduced.